adoc/admin-dr-deployment-on-aws.adoc

```diff
@@ -71,9 +71,9 @@ helm install \
 [NOTE]
 ====
 If Velero installed other than default namespace `velero`, setup velero config config to the Velero installed namespace.
-```
+----
```

Comment:
I changed this across several docs. Please use the

```
[source,bash]
----
SOURCE CODE HERE
----
```
format from now on. The "Git style" format is supported but not really ideal because it's easy to misread/mistype.

Reply:
ok